include/wx/webview.h

```diff
@@ -361,10 +365,11 @@ class WXDLLIMPEXP_WEBVIEW wxWebViewWindowInfo
 
     virtual bool ShouldDisplayScrollBars() const = 0;
 
-    virtual wxWebView* CreateChildWebView() = 0;
+protected:
+    mutable bool m_childWebViewWasUsed; // modified from const method
```

Comment:
Sorry for continuing to nitpick, but `GetChildWebView()` is not actually const, which is why I was wondering why it was mutable in the first place. Did you intend to make it const, or is there some other method modifying this variable somewhere?

Reply:
I see, I indeed intented it to be const. Will change that.